src/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "opc-plc": {
+      "commandName": "Project",
+      "commandLineArgs": "--pn=50000 --at X509Store --autoaccept --nospikes --nodips --nopostrend --nonegtrend --nodatavalues --sp --sn=25 --sr=10 --st=uint --fn=5 --fr=1 --ft=uint"
+    }
+  }
+}
```

Comment:
wieso ist das launchSettings.json hier drin? das sollte eigentlich durch .gitignore gefiltert werden.

vlt sollten wir eine sample commandline in das readme.md aufnehmen.